platform_versioned_docs/version-23.4.0/data/data-studios.mdx

```diff
@@ -102,40 +118,30 @@ Once connected, the data studio session list will update to display the status o
 
 ![](./_images/data_studios.png)
 
-### Collaborate in a data studio
+## Collaborate in a data studio
 
 To share a link to a running data studio with collaborators inside your workspace, select the three dots next to the status message for the data studio you want to share, then select **Copy data studio URL**. Using this link other authenticated users can access the session directly.
 
 :::note
 Collaborators will need valid permissions to connect to the running data studio (at least the **Connect** role for the workspace).
 :::
 
-### Stop a data studio
+## Stop a data studio
 
-To stop a running session, select the three dots next to the status message and then select **Stop**. The status will change from **running** to **stopped**. When a data studio session is stopped, the compute resources it's using are deallocated and any unsaved state is lost. You can stop a session at any time.
+To stop a running session, select the three dots next to the status message and then select **Stop**. The status will change from **running** to **stopped**. When a data studio session is stopped, the compute resources it's using are deallocated. You can stop a session at any time.
 
 :::note
 Make sure you save before you stop a data studio.
 :::
 
-### Restart a stopped data studio
+## Restart a stopped data studio
 
-When you restart a stopped session, a new session with the same configuration is created. Note that none of the data from previous sessions is preserved unless saved to the mounted data.
+When you restart a stopped session, a new session with the same configuration is created.
 
-### Delete a data studio
+## Delete a data studio
```

Comment:
```suggestion
## Start a new data studio from a checkpoint

This functionality is available to all user roles with the **Maintainer** role and above.

You can start a new data studio from an existing stopped data studio. This will inherit the history of the parent data studio checkpoint state. From the list of **stopped** data studios in your workspace, select the three dots next to the status message for the data studio and select **Start as new**. Alternatively, from the data studio detail page select the **Checkpoints** tab and choose a non-running checkpoint to start a new data studio from by clicking the three dots in the `Actions` column and selecting **Start as new**.

## Delete a data studio

This functionality is available to all user roles with the **Maintainer** role and above.
```